### ISSUE TITLE
Add publications link to main menu

### ISSUE THREAD

**Issue by wrvhage** [NLeSC/software.esciencecenter.nl#153]
Add "Publications" next to "Software", "People", etc.

**Comment by nielsdrost** [comment]
Fixed in #157

**Comment by sverhoeven** [comment]
My suggestion:
1. Add dois property to project, which contains a list of dois of publications made in the project
2. Create `estep generate publications` command, which will create a publications data file containing for each doi a record with
  * the doi
  * list of project uris (sources of the doi)
  * the html of doi converted to bibliography, using style `apa` or `ieee-with-url`, generated with `curl http://data.crossref.org/10.1016/j.daach.2016.03.001 -H 'Accept: text/bibliography; style=ieee-with-url'` for example

3. Create `publication.html` template with list of publications with a link to doi and link to it's project(s) 
4. On `_layouts/project.html`, render list of publications of the project itself

**Comment by nielsdrost** [comment]
Agreed. Do we also want to fetch metadata by getting the json version of the entry?

`curl http://data.crossref.org/10.1016/j.daach.2016.03.001 -H 'Accept:application/vnd.citationstyles.csl+json'
`

**Comment by sverhoeven** [comment]
Only if we do something with that json. Like having a filtering on publication date/author/publisher (similar to software page) or grouping per year. 

If we want that, create a separate issue for it, lets use this issue to get a simple version for publications done.

**Comment by sverhoeven** [comment]
The publications should be ordered by date.
For this we need to fetch the json.

**Comment by sverhoeven** [comment]
Added generator which fetches publication date.
```
estep generate publication http://dx.doi.org/10.1002/cpe.3416
```

**Comment by sverhoeven** [comment]
The publications with a DOI from http://research.esciencecenter.nl/ have been added as `_publication/*.md` files.

Most of the publications don't have a DOI. Their DOI's need to be retrieved and their publication Markdown files must be generated.

Also the publications should be tied to a project when possible.
